src/components/connect.js

```diff
@@ -49,6 +50,8 @@ export default function connect(mapPropsToRequestsToProps, options = {}) {
     invariant(isPlainObject(mapping), 'Request for `%s` must be either a string or a plain object. Instead received %s', prop, mapping)
     invariant(mapping.url, 'Request object for `%s` must have `url` attribute.', prop)
 
+    mapping.refreshHidden = mapping.refreshHidden === undefined ? true : mapping.refreshHidden
```

Comment:
Ignore what I said previously about moving this into `buildRequest()`. For #31, I need to break out the defaults from the request building. Once that is done, this can move there (no need to block on that in the meantime though).